src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -711,7 +711,12 @@ def _cleanup(self):
         for project_id in (set(self.ralph_projects.keys()) - set(
             self.openstack_projects.keys())
         ):
-            cloud_project = CloudProject.objects.get(project_id=project_id)
+            cloud_project = CloudProject.objects.get(
```

Comment:
This change has been merged to ng branch already in another PR. For some reason, github is still showing it in this diff.